docs/how_to_dev_frontend_en.md

```diff
@@ -0,0 +1,105 @@
+# VisualDL Web App Develop
+
+VisualDL has two major components. One is the Python/C++ SDK that logs the matrix during training,
+the other is the web-app that display visualized training data.
+
+The web-app reads the extract data and draw scalars graph or histograms graphs.
+The web-app is where VisualDL transforms the code data into graphs so users can visualize the training progress and make a better network design evaluation.
+
+This article will go over the basic web-app architecture and development steps.
+
+## Web App Overview
+The VisualDL web-app home page is constructed by three major Javascript files
+1. manifest.js: The manifest configures basic settings  
+1. vendor.js: The vendor.js stores all the 3rd party libraries.
```

Comment:
Where are the `manifest` and `vendor` files? I can't even find them

Reply:
They are under frontend/dist

Reply:
Na, these are only the built files I think. Unless you are explaining the built stuff - which should come much much later - you probably don't want to talk about these. I don't actually see them in the source repo tree too, because built assets are usually not committed. Tell me if you see something that I don't!